DOCSP-37356: Update Retrieve guide Instruqt rST (#878)
mongodb/docs-node: source/fundamentals/crud/read-operations/retrieve.txt

.. tip:: Interactive Lab
   
   This page includes a short interactive lesson that demonstrates how to

>    retrieve data by using the ``find()`` method. You can complete this lab

Reviewer: S: Since we're referring to it initially as a lesson
```suggestion
   retrieve data by using the ``find()`` method. You can complete this lesson
```

Author: ended up changing all mentions of "lesson" to "lab," since I think Instruqt is better described as a lab, but fixed!

   directly in your browser window without installing MongoDB or a code editor.

>    To start the lab, click the :guilabel:`Open Interactive Tutorial` button to the
>    right of the page header. To expand the lab to a full-screen format, click the

Reviewer: S: I think it would be sufficient and more easily understandable to say something like "at the top of the page" to help users locate the button.

```suggestion
   To start the lesson, click the :guilabel:`Open Interactive Tutorial` button at the top of the page. To expand the lesson to a full-screen format, click the
```

>    right of the page header. To expand the lab to a full-screen format, click the
>    full-screen button, :guilabel:`⛶`, in the top-right corner of the lab pane.

Reviewer: I think we would show this in the same way that we do when we describe symbols like + or = on other pages.

```suggestion
   right of the page header. To expand the lesson to a full-screen format, click the
   full-screen button (:guilabel:`⛶`), in the top-right corner of the lesson pane.
```
